alexandria/storage/src/list.cairo

```diff
@@ -116,6 +118,19 @@ impl ListImpl<
 
         popped
     }
+
+    fn array(self: @List<T>) -> Array<T> {
+        let mut array = ArrayTrait::<T>::new();
+        let mut index = 0;
+        loop {
+            if index == *self.len {
+                break ();
```

Comment:
There was a bug preventing this (so maybe still there), but you could:
```
break array;
```
 instead? and not returning it at the end of the fn

Reply:
I think I tried, but I got the following issue, I don't think it is worth it, but I could be wrong.
![image](https://github.com/keep-starknet-strange/alexandria/assets/97087040/20301a78-cc36-4902-8683-ebe19fba9514)

Reply:
y, that is what I thought :/ nvm then :)

Reply:
Are you sure it's a bug though ?

Reply:
I think so. `break`ing from a `loop` has been problematic for a long time, there were multiple bugs already reported and fixed. It works well when returning from outside of the `loop`.